src/PrestaShopBundle/Entity/Repository/StockRepository.php

```diff
@@ -194,11 +194,11 @@ private function selectStockBy(ProductIdentity $productIdentity)
      */
     public function getData(QueryParamsCollection $queryParams)
     {
-        $this->stockManager->updatePhysicalProductQuantity(
-            $this->shopId,
-            $this->orderStates['error'],
-            $this->orderStates['cancellation']
-        );
+        $product_list = parent::getData($queryParams);
```

Comment:
Not sure it's the best idea to trigger the parent call twice :thinking:

Reply:
The first parent call get the products to show in the list. After sync the products with the syncAllStock function, you must be sure that data to be showed is updated, so, a second call (parent call) is needed for this.